kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1788,4 +1788,12 @@ object KyuubiConf {
       .version("1.6.0")
       .booleanConf
       .createWithDefault(true)
+
+  val KUBERNETES_CONTEXT: OptionalConfigEntry[String] =
+    buildConf("kyuubi.kubernetes.context")
```

Comment:
`kyuubi.engine.kubernetes.context` ?  this context is for application manger, not for kyuubi server

Reply:
This ApplicationManger should correspond to Kyuubi's Kubernetes Context, means one kyuubi server => one kubernetes cluster. And Kyuubi does not currently use a similar Kubernetes Context, I think this can be reused by other Kyuubi features in the future.